src/components/form/book-submit.js

```diff
@@ -1,34 +1,53 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/booksSlice';
 
-class Form extends React.PureComponent {
-  render() {
-    return (
-      <>
-        <h2>Add New Book</h2>
-        <form action="#">
-          <div className="form-group">
-            <input
-              type="text"
-              className="form-control"
-              id="title"
-              placeholder="Title"
-            />
-          </div>
-          <div className="form-group">
-            <input
-              type="text"
-              className="form-control"
-              id="author"
-              placeholder="Author"
-            />
-          </div>
-          <button type="submit" className="btn btn-primary">
-            Add Book
-          </button>
-        </form>
-      </>
-    );
-  }
-}
+const Form = () => {
+  const dispatch = useDispatch();
+  const handleSubmit = (e) => {
+    e.preventDefault();
```

Comment:
- [ ] This component should have a local state with  `title`, `author`
- [ ] You should have your title and author inputs that update the local state every time they update with an (`onChange`) method